orbstation/species/zombie/zombie_bite.dm

```diff
@@ -0,0 +1,67 @@
+/// Time it takes to bite the target.
+#define ZOMBIE_BITE_TIME 10 SECONDS
```

Comment:
Yeah, it might be a bit long. I do want to make sure there's enough time for someone to potentially intervene, but I'll see about reducing this a bit.